### ISSUE TITLE
Can't upgrade from 138 to 140 on Huawei WS880

### ISSUE THREAD

**Issue by s10** [Jackysi/advancedtomato#345]
After pressing "Upgrade" button, upload doesn't start (no data seemed to be sent), and then after a few seconds all connections to router stop (no ping to ethernet router ip, wireless is off).

**Comment by Jackysi** [comment]
Um, so router bricked it self? I really don't know what that could be. Did you try clearing NVRAM?

**Comment by s10** [comment]
No, i didn't try clearing NVRAM. I've been upgrading advancedtomato firmware since 2015-08 without clearing NVRAM on each upgrade, it worked well for me.
If I reboot router after several minutes (5-10 or more) since pressing "Upgrade" button (after connectivity loss), it just boots with old firmware and with old configs. But if I set "clear NVRAM", i'm not sure, that router will not be bricked.

**Comment by AlexeyMoiseev** [comment]
Same problem.

**Comment by elect0nik** [comment]
Same problem

**Comment by s10** [comment]
Finally, I've updated firmware using CFE Emergency Boot page. Because even if there is a bug in 138 version upgrade page, it couldn't be fixed without firmware upgrade...

How:
1) Turn off router, press Reset button, insert power, continue to hold Reset button for 20 seconds
2) Connect to LAN2 port via ethernet cable with the static address 192.168.1.2. At the 192.168.1.1 CFE Boot upload new firmware.

If router's LAN subnet was changed, for example, to 192.168.128.0/24, then CFE boot will be available at the 192.168.128.1, not 192.168.1.1.
